pkg/document/change/change.go

```diff
@@ -54,7 +54,7 @@ func New(id ID, message string, operations []operations.Operation, p *innerprese
 // Execute applies this change to the given JSON root.
 func (c *Change) Execute(root *crdt.Root, presences *innerpresence.Map) error {
 	for _, op := range c.operations {
-		if err := op.Execute(root); err != nil {
+		if err := op.Execute(root, operations.WithVersionVector(c.ID().versionVector)); err != nil {
```

Comment:
Currently we only have a single option(`VersionVector`) which makes the pattern overhead unnecessary and JS SDK doesn't introduce option pattern so keeping the interfaces aligned would improve consistency.

So it seems that removing the Option pattern from the Operation interface is good for now. Is there a any other reason what I missing for this?

Reply:
Thanks for the review. I've modified it.